### ISSUE TITLE
Mergify not merging

### ISSUE THREAD

**Issue by eric-czech** [pystatgen/sgkit#138]
Is it normal for mergify to take ~2 hours to try to merge a PR?  This one seems stuck: https://github.com/pystatgen/sgkit/pull/114/checks?check_run_id=1021253465.  

Do you typically wait these out @jeromekelleher or is it more likely that something has gone wrong?

**Comment by tomwhite** [comment]
Also not sure what happened here: https://github.com/pystatgen/sgkit/pull/136

**Comment by ravwojdyla** [comment]
@tomwhite nothing weird happened in #134, see: https://github.com/pystatgen/sgkit/pull/136#issuecomment-679142094

**Comment by ravwojdyla** [comment]
@eric-czech FYI it looks like there is no merge queue information for #114 in the mergify dashboard: https://dashboard.mergify.io/installation/11295598/queues, so maybe it died midway 🤔 It doesn't seem like there is a lot of inspection available for what's going on inside mergify, we might need to fix this one manually, do you want me to do that @eric-czech ?

**Comment by eric-czech** [comment]
> we might need to fix this one manually, do you want me to do that @eric-czech ?

If you know how @ravwojdyla that would be awesome.  Mind documenting some general guidelines for that in case it happens again (perhaps just on this issue)?  I don't even know where to begin.

**Comment by jeromekelleher** [comment]
I think there's an issue with branch protection rules in the repo:
```
Waiting for the Branch Protection to be validated
Branch Protection is enabled and is preventing Mergify to merge the pull request. Mergify will merge when branch protection settings validate the pull request. (detail: Merge commits are not allowed on this repository.)
```

We must have either changed the config or added new protection rules recently? I'm not sure why mergify would try to merge rather than rebase.

**Comment by ravwojdyla** [comment]
@eric-czech @jeromekelleher 👍 I think that PR branch is a bit messy (master merged where it should be rebased), I think we have at least 2 options:
 1. squash merge manually right now
 2. submit PR to Timothy's repo or have Timothy clean it up

This repo has merge commit disable thus mergify can't do that, also I'm not sure why mergify would even attempt to try merge instead of the configured rebase (maybe it's a fallback method in case of rebase failure due to master merged into feature branch, which valiants breaks the official [git doc](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_public_project)).

I think we should just squash merge this manually (less work) and ok result.

**Comment by jeromekelleher** [comment]
> I think we should just squash merge this manually (less work) and ok result.

+1, that's the simplest thing to do, I think.

Seems like we do need to keep some rebasing disciple with PR branches all right.

**Comment by ravwojdyla** [comment]
Squash merged #114. Closing this.
